From eda6561e0ef6120efdca7d74b308c98ed07f8e3c Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 7 Feb 2022 17:00:44 +0200
Subject: [PATCH 44/51] s32-gen1: Remove CONFIG_SYS_FSL_ESDHC_ADDR

CONFIG_SYS_FSL_ESDHC_ADDR is not used because S32GEN1 platforms
use DM version of USDHC driver.

Issue: ALB-8484
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 include/configs/s32-gen1.h | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/include/configs/s32-gen1.h b/include/configs/s32-gen1.h
index 70f6b382ff..2e28683eda 100644
--- a/include/configs/s32-gen1.h
+++ b/include/configs/s32-gen1.h
@@ -23,13 +23,6 @@
 #define CONFIG_SYS_FSL_DRAM_SIZE2       0x80000000
 #endif
 
-/* The configurations of this board depend on the definitions in this file and
-* the ones in the header included at the end, configs/s32.h */
-
-#define CONFIG_SYS_FSL_ESDHC_ADDR	USDHC_BASE_ADDR
-
-/* #define CONFIG_CMD_EXT2 EXT2 Support */
-
 /* Ethernet config */
 
 #define CONFIG_FEC_XCV_TYPE     RGMII
-- 
2.17.1

